docs/apis/calcite.md

```diff
@@ -0,0 +1,570 @@
+Apache Apex, unified stream processing engine enables application developers to process data at very high throughput with low latency. Though type of data demands different type of
+processing requirements but SQL remains way to process input data .To ensure that existing ETL developers and developers who are well versed with Database applications adopt stream processing application development with ease, integration of SQL with Apex was needed. Being popular Apache project, Apache Calcite was chosen for an integration and to facilitate use of SQL with Apex.
```

Comment:
Order is as follows:
<End of sentence/subsentence><full stop/comma><space><start of next sentence/sub-sentence>

Please verify this throughout the document for both fullstop and comma. This is missed at a bunch of places and does not showcase a good documentation style.